server/package.json

```diff
@@ -2,8 +2,8 @@
   "name": "full-stack-template-server",
   "private": true,
   "scripts": {
-    "unit": "sh -c 'test_name=$(printf '%s' \"$0\" | sed -e 's/^sh$//g') && mocha -r ts-node/register --recursive \"**/${test_name:-*}.unit.ts\"'",
-    "test": "mocha -r ts-node/register --timeout 2000 --recursive \"**/${test_name:-*}.test.ts\"",
+    "unit": "jest src/",
+    "test": "jest test/",
```

Comment:
$test_name should be used to filter tests, so that you can also run a single test of a test suite (e.g. `taito test:server jest myTest` -> runs myTest of jest suite).